### ISSUE TITLE
Add primaryClass for bibtex format

### ISSUE THREAD

**Issue by golnazads** [adsabs/export_service#57]
```
@INPROCEEDINGS{2009arad.workE..32A,
   author = {{Accomazzi}, A. and {Murray}, S.~S. and {Kurtz}, M.~J.},
    title = "{Towards a Resource-Centric Data Network for Astronomy}",
booktitle = {``Proceedings of the special session ''Accelerating the Rate of Astronomical Discovery`` of the 27th IAU General Assembly. August 11-14 2009. Rio de Janeiro, Brazil. Published online at <A href=''http://pos.sissa.it/cgi-bin/reader/conf.cgi?confid=99``>http://pos.sissa.it/cgi-bin/reader/conf.cgi?confid=99</A>, id.32''},
     year = 2009,
archivePrefix = "arXiv",
   eprint = {1005.1886},
 primaryClass = "astro-ph.IM",
      eid = {32},
    pages = {32},
   adsurl = {http://adsabs.harvard.edu/abs/2009arad.workE..32A},
  adsnote = {Provided by the SAO/NASA Astrophysics Data System}
}
```

**Comment by golnazads** [comment]
Need a new release of adspy.

**Comment by golnazads** [comment]
adspy released 8/6/2018, verified that primaryClass made it to solr.

**Comment by golnazads** [comment]
@ARTICLE{2018arXiv180802013L,
       author = {{Li}, Yuanchun and {Yang}, Ziyue and {Guo}, Yao and {Chen}, Xiangqun and
        {Agarwal}, Yuvraj and {Hong}, Jason},
        title = "{Automated Extraction of Personal Knowledge from Smartphone Push
        Notifications}",
      journal = {ArXiv e-prints},
     keywords = {Computer Science - Information Retrieval, Computer Science - Software
        Engineering},
         year = 2018,
        month = Aug,
          eid = {arXiv:1808.02013},
        pages = {arXiv:1808.02013},
archivePrefix = {arXiv},
       eprint = {1808.02013},
 primaryClass = {cs.IR, cs.SE},
       adsurl = {https://ui.adsabs.harvard.edu/#abs/2018arXiv180802013L},
      adsnote = {Provided by the SAO/NASA Astrophysics Data System}
}

**Comment by golnazads** [comment]
thank you so much. done.
implemented in v2.0.29
